.gitignore

```diff
@@ -36,6 +36,7 @@
 .Trashes
 ehthumbs.db
 Thumbs.db
+.ruby-version
```

Comment:
Why are we gitignoring the ruby-version? We want that file to stay with the repo?